exercises/react/react_test.py

```diff
@@ -3,7 +3,7 @@
 
 from react import InputCell, ComputeCell
 
-# Tests adapted from `problem-specifications//canonical-data.json` @ v2.0.0
+# Tests adapted from `problem-specifications//canonical-data.json` @ v
```

Comment:
The blank version here is due to changes in problem-specifications; I need to make changes to the generator to account for this, and then we can update this PR.

Reply:
Test update :)